configuration_version_test.go

```diff
@@ -144,6 +144,46 @@ func TestConfigurationVersionsRead(t *testing.T) {
 	})
 }
 
+func TestConfigurationVersionsReadWithOptions(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	wTest, wTestCleanup := createWorkspaceWithVCS(t, client, orgTest, true)
+	defer wTestCleanup()
+
+	// Hack: Wait for TFC to ingress the configuration and queue a run
+	time.Sleep(3 * time.Second)
+
+	w, err := client.Workspaces.ReadByIDWithOptions(ctx, wTest.ID, &WorkspaceReadOptions{
+		Include: "current-run.configuration-version",
```

Comment:
Same as the [comment below](https://github.com/hashicorp/go-tfe/pull/232/files#r649213396) regarding using a constant.